### ISSUE TITLE
Mark as deprecated

### ISSUE THREAD

**Issue by UmamiAppearance** [TrySound/rollup-plugin-terser#123]
As the official plugin is now [available](https://github.com/rollup/plugins/tree/master/packages/terser#readme), it would be helpful to mark this repository as deprecated, and also provide a link to the replacement, to guide everyone in the right direction.

Thank you for the work you put into the package. 👍

**Comment by jonkoops** [comment]
Yeah, this is very confusing. I had this plugin installed without being aware a better and official plugin is available.
